src/PathUtil.hpp

```diff
@@ -5,6 +5,23 @@
 
 #include <string.h>
 
+#if ENABLED(TUNDRA_CASE_INSENSITIVE_FILESYSTEM)
+#if defined(_MSC_VER) || defined(TUNDRA_WIN32_MINGW)
+#define PathCompareN _strnicmp
+#define PathCompare _stricmp
+#else
+#define PathCompareN strncasecmp
+#define PathCompare strcasecmp
+#endif
+#else
```

Comment:
nitpick: add a comment to clarify this, eg `#else  // !TUNDRA_CASE_INSENSITIVE_FILESYSTEM` ?

Reply:
Fixed